apps/web/cypress/tests/activity-graph.spec.ts

```diff
@@ -1,4 +1,8 @@
-describe('Activity page', function () {
+/**
+ * The tests from this file were moved to the corresponding Playwright file apps/web/tests/activity-graph.spec.ts.
+ * @deprecated
+ */
+describe.skip('Activity page', function () {
```

Comment:
Deprecated the Cypress test files that were migrated to Playwright. Decided not to remove them for now for the review process.